### DISCUSSION TITLE
How to add tailwindCSS into cypress

### DISCUSSION THREAD

**Discussion by alamenai** [tailwindlabs/tailwindcss#12453] [Help] [answered]
Hi @adamwathan ,

Thank you so much for creating TailwindCSS.

I'm testing a component built with shadcn and styled with TaiwindCSS using Cypress.

Currently, cypress display the component without styling and I wanted to know how to add tailwindCSS classes into Cypress test.

My current testing file:

```tsx

// What should I import here to make sure that these classes works?

import { Intro } from "./Intro"

describe("", () => {
  it("", () => {
    cy.mount(<Intro />)

    cy.get("[data-cy='report-intro']").should("exist")
  })

  it("", () => {
    cy.mount(<Intro />)

    cy.get("[data-cy='report-intro']").find("div").should("exist").and("have.class", "bg-blue-50")
  })

  it("", () => {
    cy.mount(<Intro />)

    cy.get("[data-cy='report-intro']").find("[id='alert-close-icon']").click()

    cy.get("[data-cy='report-intro']").find("div").and("have.class", "bg-blue-600").should("not.exist")
  })
})

```

**Comment by alex-krasikau** [comment, marked as answer]
Hey @MenaiAla!

You need to import the TailwindCSS stylesheet in the test file or in a setup file that is included in your Cypress tests. You can import it directly from your compiled CSS file. The import statement should be placed at the top of your Cypress test file or in a common setup file.

```
import '../path/to/your/tailwind.css'; // Import TailwindCSS

import { Intro } from "./Intro"

describe("", () => {
  it("", () => {
    cy.mount(<Intro />)
    cy.get("[data-cy='report-intro']").should("exist")
  })

  // Rest of your tests
})
```

**Comment by alamenai** [comment]
Thank you @alex-krasikau ,

Yes, but to avoid importing that file in each testing file, I added it into the `component.ts` file in Cypress.
